x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ApiKeyService.java

```diff
@@ -160,7 +160,7 @@ public class ApiKeyService implements Closeable {
 
     public static final Setting<String> PASSWORD_HASHING_ALGORITHM = XPackSettings.defaultStoredHashAlgorithmSetting(
         "xpack.security.authc.api_key.hashing.algorithm",
-        (s) -> Hasher.PBKDF2.name()
+        (s) -> Hasher.SSHA256.name()
```

Comment:
There is currently no `PBKDF2` option with cost lower than `1000`. If we add a new hasher type/cost we need to worry about BWC (old node not understanding keys generated by new nodes).